src/GovUk.Education.ExploreEducationStatistics.Content.Services/DataSetService.cs

```diff
@@ -118,6 +123,57 @@ public DataSetService(ContentDbContext contentDbContext)
             })
             .ToListAsync();
     }
+
+    public async Task<Either<ActionResult, DataSetDetailsViewModel>> GetDataSet(
+        Guid releaseId,
+        Guid fileId)
+    {
+        if (!await _releaseRepository.IsLatestPublishedReleaseVersion(releaseId))
+        {
+            return new NotFoundResult();
+        }
+
+        var viewModel = await _contentDbContext.ReleaseFiles
+            .Where(rf =>
+                rf.FileId == fileId
+                && rf.ReleaseId == releaseId)
+            .Select(releaseFile => new DataSetDetailsViewModel()
+            {
+                Title = releaseFile.Name ?? "No name",
+                Summary = releaseFile.Summary ?? "No summary",
+                Release = new DataSetDetailsReleaseViewModel()
+                {
+                    Id = releaseFile.ReleaseId,
+                    Title = releaseFile.Release.Title,
+                    Slug = releaseFile.Release.Slug,
+                    Type = releaseFile.Release.Type,
+                    IsLatestPublishedRelease =
+                        releaseFile.Release.Publication.LatestPublishedReleaseId == releaseFile.ReleaseId,
+                    Published = releaseFile.Release.Published!.Value,
+                    Publication = new DataSetDetailsPublicationViewModel()
+                    {
+                        Id = releaseFile.Release.PublicationId,
+                        Title = releaseFile.Release.Publication.Title,
+                        Slug = releaseFile.Release.Publication.Slug,
+                        ThemeTitle = releaseFile.Release.Publication.Topic.Theme.Title,
+                    },
+                },
+                File = new DataSetDetailsFileViewModel()
```

Comment:
Parentheses here in `new DataSetDetailsFileViewModel()` are redundant when using object initialiser syntax.

This comment applies elsewhere in this file too.